test/UserRepository/HtpasswdFactoryTest.php

```diff
@@ -32,6 +34,29 @@ class HtpasswdFactoryTest extends TestCase
     /** @var HtpasswdFactory */
     private $factory;
 
+    /**
+     * @psalm-return Generator<string,array{0:array<string,mixed>|ArrayObject<string,mixed>, 1:non-empty-string}>
+     */
+    public function validConfigs(): Generator
+    {
+        $filename = __DIR__ . '/../TestAssets/htpasswd';
+        $config   = [
+            'authentication' => [
+                'htpasswd' => $filename,
+            ],
+        ];
+
+        yield 'array' => [
+            $config,
+            $filename,
+        ];
+
+        yield ArrayObject::class => [
```

Comment:
No point using yield, since it will be converted to array before tests are run

Reply:
I prefer yield and tests are green so actually no need to change this as we are using yield in many other packages aswell.